packages/@lwc/template-compiler/src/__tests__/fixtures/directive-render-mode/error-slots-attributes/metadata.json

```
@@ -2,7 +2,7 @@
     "warnings": [
         {
             "code": 1134,
-            "message": "LWC1134: Invalid attribute(s) 'class' on slot. Slots in Light DOM templates (which have 'lwc:render-mode' directive) can only have [name, lwc:slot-bind, key] attributes",
+            "message": "LWC1134: Invalid attribute(s) 'class' on slot. Slots in Light DOM templates (which have 'lwc:render-mode' directive) can only have [name, slot, lwc:slot-bind, key] attributes",
```

```suggestion
            "message": "LWC1134: Invalid attribute(s) 'class' on slot. Slots in Light DOM templates (which have 'lwc:render-mode' directive) can only have [key, lwc:slot-bind, name, slot] attributes",
```

Nit: let's put these in alphabetical order.